src/context.rs

```diff
@@ -381,6 +381,11 @@ impl Context {
             },
         };
 
+        let creation_pcrs = match creation_pcrs {
+            Some(val) => val,
+            None => PcrSelectionListBuilder::new().build(),
+        };
```

Comment:
```suggestion
        let creation_pcrs = creation_pcrs.unwrap_or(PcrSelectionListBuilder::new().build());
```

Reply:
Thanks for reminding me of all the alternative functions that avoid match'es that I keep forgetting to use!